_partials/_cloudtrial.md

```diff
@@ -1,5 +1,8 @@
 <Highlight type="cloud" header="Try for free on Timescale" button="Try for free">
-Your Timescale trial is completely free for you to use for the first
-thirty days. This gives you enough time to complete all the tutorials and run a
-few test projects of your own.
+<ul>
+<li> get started at the click of a button </li>
+<li> get access to advanced cloud features like transparent bottomless object storage </li>
+<li> don't waste time running high performance, highly available TimescaleDB and PostgreSQL in the cloud </li>
```

Comment:
I read this as don't waste time using Timescale. Can you rephrase please.

Reply:
This is the old one, I've actually reverted b/c my changes don't use it and it leaks out. 

We can attack CTAs in general in another PR